### PR TITLE
Update doc scripts to fix links in product docs / app tiles

### PR DESCRIPTION
**Pull request by seonsfx**

### CODE REVIEW

File: helpers/constants.py [hunk 1]
```diff
@@ -24,3 +24,38 @@
 # Skip anything other than metric docs in docs dir
 docs_to_skip = set(["readme.md", "docs-temp.md"])
 rtd_base_url = "https://docs.signalfx.com/en/latest/integrations/integrations-reference/integrations."
+
+monitor_doc_base_url = "https://docs.signalfx.com/en/latest/integrations/agent/monitors/"
+
+github_to_doc_map = {
+    "https://github.com/signalfx/integrations/tree/master/signalfx-agent": "https://docs.signalfx.com/en/latest/integrations/agent/index.html",
```
Comment (keitwb) [inline helpers/constants.py]: Do we really have to hardcode all of these links?

Comment (seonsfx) [reply helpers/constants.py]: @keitwb Few of them are inevitable but I adopted some pattern match for the rest of it.

**Comment by seonsfx** [comment]
@keitwb Should be good now. Thanks for helping

File: helpers/constants.py [hunk 2]
```diff
@@ -24,3 +24,15 @@
 # Skip anything other than metric docs in docs dir
 docs_to_skip = set(["readme.md", "docs-temp.md"])
 rtd_base_url = "https://docs.signalfx.com/en/latest/integrations/integrations-reference/integrations."
+
+monitor_doc_base_url = "https://docs.signalfx.com/en/latest/integrations/agent/monitors/"
+
+github_to_doc_map = {
```
Comment (keitwb) [inline helpers/constants.py]: Just remove this and we'll fix the links in Smart agent repo later.

**Pull request merged**